# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/libdav1d/moz.yaml
# Commit: a5108ffbd311
# Full Hash: a5108ffbd31101e7cb72dabc271d92027f953c3c
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-04-20 09:35:22
# Regressor Bug: 1540830
# File Overlap Count: 3
# Description:
#   Bug 1540830 - Update dav1d from upstream to 1f7a7e8. r=TD-Linux
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D28200
# ==============================================================================

diff -r 98adabf295d0 -r a5108ffbd311 media/libdav1d/moz.yaml
--- a/media/libdav1d/moz.yaml	Fri Apr 19 20:49:36 2019 +0000
+++ b/media/libdav1d/moz.yaml	Fri Apr 19 20:36:10 2019 +0000
@@ -20,7 +20,7 @@
 
   # Human-readable identifier for this version/release
   # Generally "version NNN", "tag SSS", "bookmark SSS"
-  release: commit 7350c59e7894cb7e487a0add9942d2b1b39f7161 (2019-03-16T23:17:05.000Z).
+  release: commit 1f7a7e8a6af739a05b320151d04f0f7509ae7579 (2019-04-19T07:16:39.000Z).
 
   # The package's license, where possible using the mnemonic from
   # https://spdx.org/licenses/